CMake/ARMA_FindACML.cmake

```diff
@@ -0,0 +1,37 @@
+# - Find AMD's ACML library (no includes) which provides optimised BLAS and LAPACK functions
```

Comment:
I'm a little lost; what's using the ARMA_*.cmake files?

Reply:
I am not sure I have take then from ensmallen repo : https://github.com/mlpack/ensmallen/tree/master/CMake

Reply:
I think you can remove them. If I'm wrong, then just revert that commit.